docs/sources/reference/commandline/cli.md

```diff
@@ -652,17 +654,20 @@ is not recursive.
     */temp*
     */*/temp*
     temp?
+    *.md
+    !LICENCE.md
 
 The first line above `*/temp*`, would ignore all files with names starting with
 `temp` from any subdirectory below the root directory. For example, a file named
 `/somedir/temporary.txt` would be ignored. The second line `*/*/temp*`, will
 ignore files starting with name `temp` from any subdirectory that is two levels
 below the root directory. For example, the file `/somedir/subdir/temporary.txt`
-would get ignored in this case. The last line in the above example `temp?`
+would get ignored in this case. The third line in the above example `temp?`
 will ignore the files that match the pattern from the root directory.
 For example, the files `tempa`, `tempb` are ignored from the root directory.
 Currently there is no support for regular expressions. Formats
-like `[^temp*]` are ignored.
+like `[^temp*]` are ignored. The second to last line adds a rule to ignore all
+markdown files and the subsequent line adds an exlusion rule for LICENCE.md.
```

Comment:
Let's add something about how order impacts it - e.g.:
```
It is important to point out that the order in which the "!" lines appear will influence the matching algorithm; the last line of the .dockerignore that matches will determine whether a file is included or not. For example, in the above example, if the last two lines were reversed then LICENSE.md would be excluded from the build because the *.md would add all "md" files back into the ignore list. In other words, the !LICENSE.md line would have no effect at all because of the subsequent *.md.
```
that is correct isn't it?